model/host/host.go

```diff
@@ -26,6 +26,9 @@ type Host struct {
 	Distro   distro.Distro `bson:"distro" json:"distro"`
 	Provider string        `bson:"host_type" json:"host_type"`
 
+	// secondary (external) identifier for the host
+	ExtIdentifier string `bson:"ext_identifier" json:"ext_identifier"`
```

Comment:
maybe name this external rather than Ext

Reply:
done